### ISSUE TITLE
Update newsletter form

### ISSUE THREAD

**Issue by Kataract** [pyohio/new-pyohio-website#10]
Is there a React version of this we can use? I dumped in the one from years past and it lacks a lot of styling, as well as pisses off `eslint-jsx-a11y` for the honeypot field.

If not then this is a task to style the form and make it look nice.
